codegenerator/cli/src/config_parsing/chain_helpers.rs

```diff
@@ -32,227 +32,304 @@ use crate::constants::DEFAULT_CONFIRMED_BLOCK_THRESHOLD;
 #[strum(serialize_all = "kebab-case")]
 #[repr(u64)]
 pub enum Network {
-    #[subenum(
-        HypersyncNetwork,
-        NetworkWithExplorer,
-        GraphNetwork(serde(rename = "mainnet"))
-    )]
-    EthereumMainnet = 1,
+    #[subenum(HypersyncNetwork, NetworkWithExplorer)]
+    Amoy = 80002,
+
+    #[subenum(NetworkWithExplorer, GraphNetwork)]
+    ArbitrumGoerli = 421613,
+
+    #[subenum(HypersyncNetwork, NetworkWithExplorer)]
+    ArbitrumNova = 42170,
+
     #[subenum(HypersyncNetwork, NetworkWithExplorer, GraphNetwork)]
-    Goerli = 5,
```

Comment:
Goerli didn't make sense to be second, I get having eth 1st as its popular but I don't think we should be opinionated on network popularity. Sensible to be alphabetical to me